### PR TITLE
Add config info to setup page for post

### PR DESCRIPTION
**Pull request by codebard**
**Problem**

The config collected from WP install by the function in add_setup_info_collector needed to be added to the setup wizard initial screen for _POST to Patreon.

**Solution**

Code was added to get the output of the setup info collector and put it as hidden input to the form in the setup wizard's initial screen.

**Verification**

Form fields can be observed by viewing the source or showing form fields via web browser developer addons. Tested in easy_setup_wizard_pre_pr_test_branch branch. 

**Does this need tests**

no

### CODE REVIEW

File: assets/css/admin.css [hunk 1]
```diff
@@ -147,4 +147,26 @@
 	left: 2px;
 	width: 23px;
 	height: 37px;
-}
\ No newline at end of file
+}
+
+#patreon_setup_screen {
```
Comment (phildini) [inline assets/css/admin.css]: I think this code is also in #79 -- is that intentional?

Comment (phildini) [inline assets/css/admin.css]: I think this code is also in #79 -- is that intentional?

Comment (codebard) [reply assets/css/admin.css]: I think 80 and 81 (this) respectively. I likely put them there due to a need to display the interface right during testing. We can remove them from one for consistency.

Comment (codebard) [reply assets/css/admin.css]: Removed.

Comment (codebard) [reply assets/css/admin.css]: Removed.

**Pull request closed without merging**